guides-and-standards/standards--source-control.md

```diff
@@ -0,0 +1,153 @@
+
+If the code in a repository requires sensitive information to run, it should be stored in a separate configuration file that is explicitly not included in the repository by adding its name to the `.gitignore` file.
+
+### Review templates
+
+It's a best practice to provide a default template for the developer to fill out when creating the pull request. This can be done by creating a file called `PULL_REQUEST_TEMPLATE.md` under the `.github` directory of your project. An example pull request template file can be found [here](../.github/PULL_REQUEST_TEMPLATE.md).
+
+### [Optional] Reviewing Jupyter notebooks using the ReviewNB tool
+
+Jupyter notebooks are great data science tools as they combine code, output, and written notes into a single document. They support code written in Python, R, and Julia, and can be coaxed to execute bash as well. Given that they are natively rendered by GitHub, they are a good option to record computational experiments. However, in a pull request, notebooks quickly become unwieldy because they include not only the code you wrote and ran but also code that specifies the format of the document. This makes wading through a diff painful. We have enabled the [ReviewNB tool](https://www.reviewnb.com/) to produce better diffs from Jupyter notebooks.
+
+## Data management
+
+- For publicly available data, scripts used to download and process these data should be preserved, as should the versions of items used in processing (e.g. probe to gene mappings). This code should be version controlled. Where possible, intermediate files of reasonable size can be stored to facilitate re-use, but the process to regenerate these files from publicly available data should be preserved.
+- Files under 100MB in size can be stored in the repository.
+- Private files over 100MB should be stored in a cloud storage service (e.g. an S3 bucket or Google Drive) and instructions to access the files should be included in the README.
+- Public files should be stored in a public archive (e.g. the Sequence Read Archive), in a Zenodo repository, or in a publicly-accessible S3 bucket. Reach out to the software team for help with this.
+- When we generate data, they should be stored in a location where they are replicated and uploaded to the relevant database as soon as possible (e.g. raw sequencing data uploaded to the Sequence Read Archive). If possible, the code should be updated to match the above. For more on this, please see the relevant publishing documentation [here](https://www.notion.so/arcadiascience/Sharing-Data-eca60da150f04a7a905a8540871f0364).
+
+## Guidelines for collaborating on codebases with collaborators who are not Arcadians
+
+- Before beginning a collaboration, please consult with your manager and our legal counsel to make sure there are not licensing and legal problems with your collaboration.
```

Comment:
Do we have legal counsel anymore?

Reply:
There was some mention of this in the most recent goals/status update. I suppose under publishing 2.0 we are our own counsel, if nothing else. Will leave as-is for now.